services/connected-status/README.md

```diff
@@ -0,0 +1,11 @@
+# DHIS2 App Connected Status
+
+Application configuration for [DHIS2](https://dhis2.org) applications
+
+This library is intended for use with the [DHIS2 Application Platform](https://github.com/dhis2/app-platform).
+
+## Installation
+
+This package is internal to `@dhis2/app-runtime` and generally should not be installed independently.
```

Comment:
This is a more a general query than something for the scope of this PR but I notice in a few of the read me files for the services we say..

This package is internal to `@dhis2/app-runtime` and generally should not be installed independently.

 If this is the case do we really want to be publishing the packages for the services themselves?  `@dhis2/app-service-connected-status` will likely get published and others like `@dhis2/app-service-alerts`  are already being published as well. Is there a use case I'm not aware of for doing this?

Reply:
I agree that one the one hand publishing a package, and on the other hand advising against using it, is confusing. IMO we should keep publishing the packages, and rephrase the sentence to something like this:
> This package also comes bundled with `@dhis2/app-runtime` and generally does not have to be installed independently.

The reason why one would want to install a package like this independently is for non-platform apps. For whatever reason, apps may need to use a different setup and we still want them to be able to use all of the providers/services we offer.

Reply:
Cool @HendrikThePendric I'll change the wording like you suggest there

Reply:
As this file has been deleted, I'll look at raising another PR to introduce this change to the other readme files